k8s/object.bzl

```diff
@@ -133,6 +133,9 @@ def _resolve(ctx, string, output):
   )
 
 def _common_impl(ctx):
+  if ctx.attr.namespace and ctx.attr.namespace_file:
```

Comment:
Validation / failure should be done in macro: https://github.com/bazelbuild/rules_k8s/blob/master/k8s/object.bzl#L416

Reply:
Will fix, thank you!

Reply:
Fixed